### DISCUSSION TITLE
Receive Parameters in Modal

### DISCUSSION THREAD

**Discussion by cainydev** [orchidsoftware/platform#2172] [Q&A]
Dear Orchid Community,

how can I receive Parameters in my Modal? I am sending the data using the fluent parameters method:

```php
ModalToggle::make()
      ->modal('deleteSupplier')
      ->class('btn btn-danger p-2')
      ->parameters(['supplier' => $supplier])
      ->icon('trash')
```

#### Is it even possible to send the full eloquent model or do I have to send the id?

I want the data in my modal itself, as well as in the action method for that modal.

_Thanks in advance, John_

**Comment by MichelNZOLO** [comment]
https://github.com/orchidsoftware/platform/discussions/2172
